src/infobox.tsx

```diff
@@ -34,10 +33,6 @@ class Main extends React.Component<any, any> {
    */
   public constructor(props: MainProps) {
```

Comment:
Maybe we can remove the constructor completely, no?

Reply:
@diocas we can't remove it, I need it to inject file information into react props

Reply:
But isn't the parent constructor already receiving that information? Usually you only redefine a method if you need to add extra logic, but here you just call the parent.